src/aggregator/server/m3msg/server.go

```diff
@@ -62,7 +63,8 @@ type messageProcessor struct {
 }
 
 func (m *messageProcessor) Process(msg consumer.Message) {
-	if err := m.handleMessage(&m.pb, &m.union, msg); err != nil {
+	if err := m.handleMessage(&m.pb, &m.union, msg); err != nil && rand.Float32() < 0.001 {
```

Comment:
Do we have other noisy error messages?

Reply:
nope, from elk, it all comes from this line: [elk link](https://kibana-aws-us-west-2.cloud.databricks.com/app/discover?security_tenant=service_logs#/?_g=(filters:!(('$state':(store:globalState),meta:(alias:!n,disabled:!f,index:'942720c0-23e4-11eb-8e4e-39c23d34e603',key:tags.subDir,negate:!f,params:(query:m3aggregator),type:phrase),query:(match_phrase:(tags.subDir:m3aggregator)))),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(columns:!(messageText),filters:!(),index:'942720c0-23e4-11eb-8e4e-39c23d34e603',interval:auto,query:(language:kuery,query:'NOT%20%22could%20not%20process%20message%22'),sort:!()))